compiler/backend/presLangScript.sml

```diff
@@ -234,381 +234,341 @@ val exh_to_pres_exp_def = tDefine"exh_to_pres_exp"`
     (exh_to_pres_pat p, exh_to_pres_exp e)::exh_to_pres_pes pes)`
   cheat; 
 
-(* pres_to_json *)
-(* TODO: Add words *)
-val lit_to_value_def = Define`
-  (lit_to_value (IntLit i) = Int i)
-  /\
-  (lit_to_value (Char c) = String [c])
-  /\
-  (lit_to_value (StrLit s) = String s)
-  /\
-  (lit_to_value _ = String "word8/64")`;
+(* Helpers for converting pres to structured. *)
+val string_to_structured_def = Define`
+  string_to_structured s = Item NONE s []`;
 
-(* Create a new json$Object with keys and values as in the tuples. Every object
-* has constructor name field, cons *)
-val new_obj_def = Define`
-  new_obj cons fields = json$Object (("cons", String cons)::fields)`;
+val num_to_structured_def = Define`
+  num_to_structured n = string_to_structured (num_to_str n)`;
 
-val num_to_json_def = Define`
-  num_to_json n = Int (int_of_num n)`;
-
-val trace_to_json_def = Define`
-  (trace_to_json (backend_common$Cons tra num) =
-    new_obj "Cons" [("num", num_to_json num); ("trace", trace_to_json tra)])
-  /\
-  (trace_to_json (Union tra1 tra2) =
-    new_obj "Union"
-      [("trace1", trace_to_json tra1); ("trace2", trace_to_json tra2)])
+val word_size_to_structured_def = Define`
```

Comment:
A function like `new_empty_item` is what I initially was thinking of, but then I saw that `string_to_structured` does exactly what I thought such a function would do.

The possible gain would be if we decide that we want to change how "bare"/"empty"/"simple"/whatever items are rendered to structuredLang (and eventually to JSON), it would be more convenient (and less error-prone) to only change that in one place, instead of for every empty constructor. The drawback, as you mentioned, would be that it is another level of indirection.